docs/script-catalog/person_authentication/fido2-external-authenticator/README.md

```diff
@@ -6,20 +6,20 @@
 FIDO2 is comprised of the [W3C’s Web Authentication specification (WebAuthn)](https://www.w3.org/TR/webauthn/) and FIDO’s corresponding [Client-to-Authenticator Protocol (CTAP)](https://fidoalliance.org/specs/fido-v2.0-ps-20170927/fido-client-to-authenticator-protocol-v2.0-ps-20170927.html). WebAuthn defines a standard web API that can be built into browsers and related web platform infrastructure to enable online services to use FIDO Authentication. CTAP enables external devices such as mobile handsets or FIDO Security Keys to work with WebAuthn and serve as authenticators to desktop applications and web services.
 
 This document explains how to use the Janssen Auth Server's built-in 
-[FIDO2 interception script](https://github.com/JanssenProject/jans/blob/vreplace-janssen-version/jans-linux-setup/jans_setup/static/extension/person_authentication/Fido2ExternalAuthenticator.py)
+[FIDO2 interception script](https://github.com/JanssenProject/jans/blob/v1.0.16/docs/script-catalog/person_authentication/fido2-external-authenticator/Fido2ExternalAuthenticator.py)
```

Comment:
You will break the auto doc generator and the docs will be outdated. Please do not remove the special keywords like `vreplace-janssen-version` with any specific version in head. This goes for there other locations in which this has occurred